### PR TITLE
Add SSL/TLS

### PR DESCRIPTION
**Pull request by Lodeiro0001**

### CODE REVIEW

File: mqtt2notifysend.py [hunk 1]
```diff
@@ -4,23 +4,36 @@
...
+BROKER = "192.168.X.XXX"
+CLIENT_NAME = "your_computer_name"
+USERNAME = "your_username"
+PASSWORD = "ypur_password"
+CLEAN_SESSION = True
+USER_DATA = None
+PROTOCOL = mqtt.MQTTv311
+TOPIC_SUB = "dev/"+CLIENT_NAME+"/toast"
+TOPIC_STAT = "dev/"+CLIENT_NAME+"/stat"
+PORT = 8883
```
Comment (Pythoneiro) [inline mqtt2notifysend.py]: Port should be defaulted to 1883

File: mqtt2notifysend.py [hunk 2]
```diff
@@ -4,23 +4,36 @@
...
 ICON = os.path.join(CURRENT_DIR, "icon.png")
 
 
-def on_connect(client: mqtt.Client, userdata, flags, rc):
-    client.subscribe(TOPIC_SUB)
-    client.publish(TOPIC_STAT, PAYLOAD_ON, retain=True)
+def on_connect(mqtt_client: mqtt.Client, userdata, flags, rc):
+    mqtt_client.subscribe(TOPIC_SUB)
+    mqtt_client.publish(TOPIC_STAT, PAYLOAD_ON, retain=True)
+    mqttClient.will_set(TOPIC_STAT, PAYLOAD_OFF, retain=True)
```
Comment (Pythoneiro) [inline mqtt2notifysend.py]: will_set must be applied BEFORE connecting

File: mqtt2notifysend.py [hunk 3]
```diff
@@ -30,23 +43,12 @@ def on_message(client: mqtt.Client, userdata, msg: mqtt.MQTTMessage):
...
 mqttClient.on_message = on_message
-mqttClient.will_set(TOPIC_STAT, PAYLOAD_OFF, retain=True)
 mqttClient.connect(BROKER, PORT, KEEPALIVE)
 
-if __name__ == "__main__":
-    try:
-        mqttClient.loop_forever()
-    except KeyboardInterrupt:
-        mqttClient.loop_stop()
+mqttClient.loop_forever()
```
Comment (Pythoneiro) [inline mqtt2notifysend.py]: a main is more canonical :smile_cat:

File: mqtt2notifysend.py (hunk 1 above)
Comment (David-Lor) [inline mqtt2notifysend.py:20]: Port should be defaulted to 1883

File: mqtt2notifysend.py (hunk 2 above)
Comment (David-Lor) [inline mqtt2notifysend.py:34]: will_set must be applied BEFORE connecting

File: mqtt2notifysend.py [hunk 4]
```diff
@@ -4,23 +4,36 @@
...
 
 
-def on_connect(client: mqtt.Client, userdata, flags, rc):
-    client.subscribe(TOPIC_SUB)
-    client.publish(TOPIC_STAT, PAYLOAD_ON, retain=True)
+def on_connect(mqtt_client: mqtt.Client, userdata, flags, rc):
+    mqtt_client.subscribe(TOPIC_SUB)
+    mqtt_client.publish(TOPIC_STAT, PAYLOAD_ON, retain=True)
+    mqttClient.will_set(TOPIC_STAT, PAYLOAD_OFF, retain=True)
+    # Comment the following line if you do not use SSL/TLS connection with the Broker
```
Comment (David-Lor) [inline mqtt2notifysend.py:35]: USERNAME and PASSWORD variables can be empty strings or None if the user won't want to use authentication

File: mqtt2notifysend.py (hunk 3 above)
Comment (David-Lor) [inline mqtt2notifysend.py:54]: a main is more canonical :smile_cat:

File: mqtt2notifysend.py (hunk 2 above)
Comment (Lodeiro0001) [reply mqtt2notifysend.py:34]: There works LIKE A CHARM! 🦓

File: mqtt2notifysend.py (hunk 3 above)
Comment (Lodeiro0001) [reply mqtt2notifysend.py:54]: you are very canonical 🐕

**Pull request closed without merging**